client/src/components/Form/form.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import Input from "../LoginForm/loginInput.js/input";
+import Joi from "joi-browser";
```

Comment:
Worth considering, not a blocker at all: https://bundlephobia.com/result?p=joi-browser@13.4.0